### PR TITLE
Language update suggestions for basic HTML curriculum

### PR DESCRIPTION
**Pull request by schalkneethling**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description
tl;dr Small language update suggestions the the curriculum for basic HTML.

I am working through the freeCodeCamp course along with my two daughters. As I find language I feel can be improved, I will be opening pull requests with said suggestions.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 1]
```diff
@@ -183,8 +183,8 @@
...
         "<a href=\"https://i.imgur.com/jOc1JF1.png\" target=\"_blank\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" src=\"https://i.imgur.com/jOc1JF1.png\" title=\"Click to enlarge\" alt=\"A screen shot of our finished Cat Photo App\"></a>",
-        "The <code>h2</code> element you enter will create an <code>h2</code> element on the website.",
-        "This element tells the browser about the structure of your website. <code>h1</code> elements are often used for main headings, while <code>h2</code> elements are generally used for subheadings. There are also <code>h3</code>, <code>h4</code>, <code>h5</code> and <code>h6</code> elements to indicate different and new sections.",
+        "The <code>h2</code> element you will be adding in this step will add a level two heading to the HTML website.",
+        "This element tells the browser about the structure of your website. <code>h1</code> elements are often used for main headings, while <code>h2</code> elements are generally used for subheadings. There are also <code>h3</code>, <code>h4</code>, <code>h5</code> and <code>h6</code> elements to indicate different levels of hierarchy.",
```
Comment (Bouncey) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: This element tells the browser about the structure of your website. <code>h1</code> elements are often used for main headings, while <code>h2</code> elements are generally used for subheadings. There are also <code>h3</code>, <code>h4</code>, <code>h5</code> and <code>h6</code> elements to indicate different levels of ~hierarchy~.

This element tells the browser about the structure of your website. <code>h1</code> elements are often used for main headings, while <code>h2</code> elements are generally used for subheadings. There are also <code>h3</code>, <code>h4</code>, <code>h5</code> and <code>h6</code> elements to indicate different levels of subheadings.

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 2]
```diff
@@ -376,9 +376,9 @@
       "id": "bad87fee1348bd9aedf08802",
       "title": "Uncomment HTML",
       "description": [
-        "Commenting is a way that you can leave comments within your code without affecting the code itself.",
+        "Commenting is a way that you can leave comments for other developers within your code without affecting the resulting output that is displayed the the end user.",
         "Commenting is also a convenient way to make code inactive without having to delete it entirely.",
-        "You can start a comment with <code>&#60;!--</code> and end a comment with <code>--&#62;</code>",
+        "Comments in HTML starts with <code>&#60;!--</code>, and you close a comment with <code>--&#62;</code>",
```
Comment (Bouncey) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Should be either open/close or start/end.

**Review by Bouncey — CHANGES_REQUESTED** [review]
Thanks for the suggestions @schalkneethling 

Seeing as these are the very few set of challenges, we try to keep the language simple but correct, your suggestions helped with this.

**Comment by schalkneethling** [comment]
Thanks for the review @Bouncey - updated.

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Great work @schalkneethling, thanks for the quick turn around 👍 

Just waiting for the CI to finish then I will merge

**Comment by Bouncey** [comment]
Thank you for your first contribution to freeCodeCamp @schalkneethling 👍
